### ISSUE TITLE
Color contrast lacking sufficient contrast ratio

### ISSUE THREAD

**Issue by carinaluise** [substrate-developer-hub/substrate-docs#1302]
### Accessibility Report
WebAIM contrast ratio tests checks on pages fails . Low-contrast text can make it difficult for many users to read.

Color Issues
#24CC85 against a #FFFFF background (Fails all checks)
#4E73F2 against a #FFFFFF background (Fails the normal text check)
WebAIM contrast checker: https://webaim.org/resources/contrastchecker/
Examples

<img width="690" alt="Main Docs" src="https://user-images.githubusercontent.com/60577717/180818132-95891471-0a25-4e39-a235-9faf33f9744f.png">

<img width="676" alt="176180583-c6ca3d2a-5fed-4d57-b50e-bbdb2397d471" src="https://user-images.githubusercontent.com/60577717/180818166-5f69f508-411d-4082-ad0d-da7da5a73bfa.png">

**Comment by jonnysmillie** [comment]
Thanks for this Carina.

These are the Substrate colours so I may need to go back to the design team with these for suggestions.

**Comment by carinaluise** [comment]
Sweet! Thanks Jonny!
